### DISCUSSION TITLE
Please help me for game collection

### DISCUSSION THREAD

**Discussion by markxiaotao** [NAIST-SE/AttractingContributors#7] [General]
https://github.com/markxiaotao/AttractingContributors-Game

**Comment by raux** [comment]
I have sent a Pull Request. Please check

**Comment by noppadol-assava** [comment]
I commit pull request on your repo.
Please help me adding Food into the list.
https://github.com/NoppadolAssava/AttractingContributors---Food

**Comment by mailtoy** [comment]
Please check my pull request :)
Add this https://github.com/mailtoy/AttractingContributors-Movies my repo, please help me.

**Comment by haxhimitsu** [comment]
I already sent pull request.

Thanks your merge:)

**Comment by nosfill** [comment]
I have sent a PR.

**Comment by satokibi44** [comment]
I have already sent PR.

**Comment by wowry** [comment]
I have sent a PR
